platform/core/src/services/DisplaySetService/DisplaySetMessage.ts

```diff
@@ -0,0 +1,87 @@
+const CODES = {
+  NO_VALID_INSTANCES: 1,
+  NO_POSITION_INFORMATION: 2,
+  NOT_RECONSTRUCTABLE: 3,
+  MULTIFRAME_NO_PIXEL_MEASUREMENTS: 4,
+  MULTIFRAME_NO_ORIENTATION: 5,
+  MULTIFRAME_NO_POSITION_INFORMATION: 6,
+  MISSING_FRAMES: 7,
+  IRREGULAR_SPACING: 8,
+  INCONSISTENT_DIMENSIONS: 9,
+  INCONSISTENT_COMPONENTS: 10,
+  INCONSISTENT_ORIENTATIONS: 11,
+  INCONSISTENT_POSITION_INFORMATION: 12,
+};
+
+const STRINGS = {
```

Comment:
I am not sure defining the strings here is best. We should run this by others, but I think we should just define the codes here and in a new locale file (e.g. `platform\i18n\src\locales\en-US\Messages.json`) add the messages where the key is the code. That way the messages can be properly translated. `Messsages.json` might look something like the following...

```
  "1": "No valid instances found in series.",
  "2": "Series has missing position information.",
  ... etc
```

Then every individual UI that wants to display a message can use their regular pattern to translate the message.

```
  const { t } = useTranslation('Messages');
  t(displaySet.messsages[0].code);
```

Reply:
Done

Reply:
I think the use of strings is fine and is in harmony with the rest of the i18n that we have